lib/server-utils.ts

```diff
@@ -257,12 +258,12 @@ export interface CustomGuiActionPayload {
     controlIndex: number;
 }
 
-export async function runCustomGuiAction(hermione: Hermione, {customGui}: ReporterConfig, payload: CustomGuiActionPayload): Promise<void> {
+export async function runCustomGuiAction(testplane: Testplane, {customGui}: ReporterConfig, payload: CustomGuiActionPayload): Promise<void> {
     const {sectionName, groupIndex, controlIndex} = payload;
     const ctx = customGui[sectionName][groupIndex];
     const control = ctx.controls[controlIndex];
 
-    await ctx.action({hermione, control, ctx});
+    await ctx.action({testplane, hermione: testplane, control, ctx});
```

Comment:
`testplane` and `hermione` props are `Object.is` copies for backward compatibility